projects/goharbor/harbor/Makefile

```diff
@@ -56,7 +56,7 @@ FIX_LICENSES_XEIPUUV_TARGET=$(REPO)/vendor/github.com/xeipuuv/gojsonpointer/LICE
 
 HAS_HELM_CHART=true
 
-HELM_GIT_TAG=v1.9.0
+HELM_GIT_TAG?=$(shell cat HELM_GIT_TAG)
```

Comment:
small thing but ive been trying to use `:=` in cases like this since every time it is accessed the file is read if you use ?=.

Reply:
done